bin/varnishncsa/varnishncsa.c

```diff
@@ -993,44 +1046,44 @@ dispatch_f(struct VSL_data *vsl, struct VSL_transaction * const pt[],
 				skip = 1;
 				break;
 			case SLT_PipeAcct:
-				frag_fields(0, b, e,
+				frag_fields(FMTPOL_FIRST, b, e,
 				    3, &CTX.frag[F_I],
 				    4, &CTX.frag[F_O],
 				    0, NULL);
 				break;
 			case SLT_BackendOpen:
-				frag_fields(1, b, e,
+				frag_fields(FMTPOL_ALWAYS, b, e,
 				    3, &CTX.frag[F_h],
 				    0, NULL);
 				break;
```

Comment:
I think FMTPOL_ALWAYS should go away, %h is listed in the first-match "internal" category. This is the only place where we use it.